packages/client-search/src/types/SearchResponse.ts

```diff
@@ -38,4 +38,50 @@ export type SearchResponse<TObject = {}> = {
   readonly parsedQuery?: string;
   readonly userData?: any;
   readonly appliedRules?: ReadonlyArray<Readonly<Record<string, any>>>;
+
+  /**
+   * The explanation of the decompounding at query time.
+   */
+  readonly explain?: {
+    /**
+     * The explain query match.
+     */
+    readonly match: {
```

Comment:
Explain is optional, but when exists, the it will **always** the following **REQUIRED** structure: